src/ui/found-at-tooltip.jsx

```diff
@@ -10,16 +10,21 @@ class FoundAtTooltip extends React.PureComponent {
 
     const sortedLocations = _.sortBy(
       locations,
-      ({ generalLocation, detailedLocation }) => {
+      (location) => {
+        const { generalLocation, detailedLocation } = location;
         const sphereForLocation = spheres.sphereForLocation(generalLocation, detailedLocation);
 
+        _.set(location, 'sphere', sphereForLocation);
+
         return _.isNil(sphereForLocation) ? Number.MAX_SAFE_INTEGER : sphereForLocation;
       },
     );
 
-    const locationsList = _.map(sortedLocations, ({ generalLocation, detailedLocation }) => (
+    const locationsList = _.map(sortedLocations, (
+      { generalLocation, detailedLocation, sphere },
+    ) => (
       <li key={`${generalLocation}-${detailedLocation}`}>
```

Comment:
Could you update this to retrieve the sphere inside the `_.map` instead of the `_.sortBy`? That means we need to call `sphereForLocation` twice, but that's not a big deal, and it allows us to avoid side effects while sorting.